tuna/miopen/subcmd/export_db.py

```diff
@@ -62,11 +62,15 @@ def get_filename(arch, num_cu, filename, ocl, db_type):
   tuna_dir = f'tuna_{version}'
   if not os.path.exists(tuna_dir):
     os.makedirs(tuna_dir)
-  final_name = f"{tuna_dir}/{arch}_{num_cu}"
-  if num_cu > 64:
-    final_name = f'{tuna_dir}/{arch}{num_cu:x}'
+
   if filename:
     final_name = f'{tuna_dir}/{filename}'
+  elif not num_cu:
```

Comment:
should probably default num_cu=None in the function definition if we expect it to be possibly None. Same with filename?

Reply:
changed